### ISSUE TITLE
InitTextures - Misaligned read error

### ISSUE THREAD

**Issue by thekovic** [jnmartin84/64doom#20]
After building the ROM with the Makefile (so far, I tried Ultimate DOOM and DOOM 2), and launching, the game goes into "LibDragon inspector" after the InitTextures function call.

![image](https://github.com/jnmartin84/64doom/assets/72971433/54c8f4d9-e743-4a7c-8349-1daea881cc18)

Built on latest unstable branch libdragon.

**Comment by jnmartin84** [comment]
I know what that was from but did you figure out why it happened?

**Comment by jnmartin84** [comment]
Also looks like I need to update libdragon again that's much better than the exception handler in the version I'm on

**Comment by jnmartin84** [comment]
I still build stack traces by hand in MAME debugger 🫠

**Comment by thekovic** [comment]
> I know what that was possibly from (W_CacheLumpName) but did you figure out why it happened?

Yeah, I closed the issue because the error was actually my fault. The game is looking for DOOM2.WAD (or any other IWAD) but I provided a WAD with name that wasn't in caps (doom2.wad). I thought about whether I should somehow try and add support for both lower and upper case but then I checked that the digital versions of all the DOOM games come with IWADs that have names in all caps, even on Windows. It's literally just the old retail copies that I've had for ages and that I use for all my source ports that for some reason have names in lower case.
